### PR TITLE
feat(app): Display content size instead of validator summary size for datasets

### PR DESCRIPTION
**Pull request by nellh**
Uses the new API implemented in #2578 to show a better estimate of dataset size in bytes.

### CODE REVIEW

File: packages/openneuro-components/src/search-page/SearchResultItem.tsx [hunk 1]
```diff
@@ -251,8 +250,7 @@ export const SearchResultItem = ({
     <Tooltip
       tooltip="Visable to all viewers"
       flow="up"
-      className="result-icon result-publlic-icon"
-    >
+      className="result-icon result-publlic-icon">
```
Comment (effigies) [inline packages/openneuro-components/src/search-page/SearchResultItem.tsx:253]: Just noticing this is "publlic" (two `l`s). I'm not finding anywhere else in the repo that uses "result-publlic-icon" or "result-public-icon", so unclear if this is a typo (and if so, if it has any meaningful effect).

Comment (nellh) [reply packages/openneuro-components/src/search-page/SearchResultItem.tsx:253]: Probably a class in the previous design that can just be removed.

File: packages/openneuro-app/src/scripts/dataset/snapshot-container.tsx [hunk 2]
```diff
@@ -111,7 +111,7 @@ const SnapshotContainer: React.FC<SnapshotContainerProps> = ({
           <div className="col col-8 col-lg">
             {summary && (
               <DatasetHeaderMeta
-                size={summary.size}
+                size={snapshot.size}
                 totalFiles={summary.totalFiles}
```
Comment (effigies) [inline packages/openneuro-app/src/scripts/dataset/snapshot-container.tsx:115]: Need `snapshot.totalFiles`.

File: packages/openneuro-components/src/search-page/SearchResultItem.tsx [hunk 3]
```diff
@@ -164,7 +165,7 @@ export const SearchResultItem = ({
   const size = (
     <span className="result-summary-meta">
       <strong>Size: </strong>
-      <span>{bytes(summary?.size)}</span>
+      <span>{bytes(node?.latestSnapshot?.size)}</span>
```
Comment (rwblair) [inline packages/openneuro-components/src/search-page/SearchResultItem.tsx]: For the curious reader:
```
> bytes(x?.bad)
null
```

Comment (effigies) [reply packages/openneuro-components/src/search-page/SearchResultItem.tsx]: Good `Maybe` monad.

Comment (nellh) [reply packages/openneuro-components/src/search-page/SearchResultItem.tsx]: Nice, so that's mostly fine (it will render "Size: ").

Comment (effigies) [reply packages/openneuro-components/src/search-page/SearchResultItem.tsx]: Would this be a reasonable failure mode?

```suggestion
      <span>{bytes(node?.latestSnapshot?.size) || "unknown" }</span>
```

**Pull request merged**